firefox-ios/Client/Frontend/Autofill/Address/AddressAutofillSettingsView.swift

```diff
@@ -37,12 +37,17 @@ struct AddressAutofillSettingsView: View {
                     .padding(.top, 25)
                     .frame(maxWidth: .infinity)
 
-                // Address list view
-                AddressListView(windowUUID: windowUUID, viewModel: addressListViewModel)
+                if addressListViewModel.showSection || addressListViewModel.isEditingFeatureEnabled {
+                    // Address list view
```

Comment:
small nit, I know it's not related to this PR but this comment is not really helpful so we can remove it.